content/id/docs/concepts/workloads/pods/pod-preset.md

```diff
@@ -0,0 +1,56 @@
+
+1. Mengambil semua `PodPreset` yang tersedia untuk digunakan.
+2. Cek jika _label selector_ dari salah satu `PodPreset` cocok dengan _label_ pada _pod_ yang sedang diciptakan.
+3. Usaha untuk menggabungkan berbagai _resource_ didefinisikan oleh `PodPreset` ke dalam _Pod_ yang sedang diciptakan.
+4. Ketika terjadi galat, lempar sebuah _event_ yang mendokumentasikan galat penggabungan dalam _pod_, dan membuat _pod_ tanpa salah satu _resource_ dari `PodPreset`.
+5. Anotasikan hasil spesifikasi _Pod_ yang telah dimodifikasi untuk menunjukkan bahwa _Pod_ telah dimodifikasi oleh sebuah `PodPreset`. Anotasi berupa `podpreset.admission.kubernetes.io/podpreset-<nama pod-preset>: "<versi resource>"`.
+
+Tiap _Pod_ akan bisa dipasangkan oleh nol atau lebih `Pod Preset`; dan tiap `PodPreset` bisa diterapkan ke nol atau lebih _pod_. Ketika sebuah `PodPreset` diterapkan ke satu atau lebih _Pod_, _Kubernetes_ memodifikasi spesifikasi _Pod_.
+
+
+>Catatan: Sebuah _Pod Preset_ mampu memodifikasi kolom `.spec.containers` pada sebuah spesifikasi *Pod* jika sesuai. Tidak ada definisi resource dari *Pod Preset* yang akan diterapkan kepada kolom `initContainer`.
+
+
+### Menonaktifkan **Pod Preset** untuk sebuah _Pod_ Spesifik
+Mungkin akan ada keadaan dimana kamu menginginkan sebuah _Pod_ tidak bisa diubah oleh sebuah mutasi _Pod Preset_. Pada kasus ini, kamu bisa menambahkan sebuah anotasi pada spesifikasi _Pod_ dalam bentuk: `podpreset.admission.kubernetes.io/exclude: "true"`.
+
+## Mengaktifkan **Pod Preset**
+---
+Dalam rangka untuk menggunakan _Pod Preset_ di dalam kluster kamu, kamu harus memastikan hal berikut:
+
+1. Kamu telah mengaktifkan tipe API `settings.k8s.io/v1alpha1/podpreset`. Sebagai contoh, ini bisa dilakukan dengan menambahkan `settings.k8s.io/v1alpha1=true` di dalam opsi `--runtime-config` untuk API _server_. Dalam _minikube_ tambahkan argumen berikut `--extra-config=apiserver.runtime-config=settings.k8s.io/v1alpha1=true` saat memulai kluster.
+
+2. Kamu telah mengaktifkan _admission controller_ dari *PodPreset*. Salah satu cara untuk melakukannya adalah dengan menambakan *PodPreset* di dalam nilai opsi `--enable-admission-plugins` yang dispesifikasikan untuk API _server_. Dalam _minikube_ tambahkan argumen berikut `--extra-config=apiserver.enable-admission-plugins=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,PodPreset ` saat memulai kluster.
+
+3. Kamu telah mendefinisikan _Pod Preset_ kamu dengan membuat objek `PodPreset` pada _namespace_ yang akan kamu gunakan. 
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+  * [Memasukkan data ke dalam sebuah _Pod_ dengan _PodPreset_](/docs/concepts/workloads/pods/pod/#injecting-data-into-a-pod-using-podpreset.md)
```

Comment:
should just be Pod and PodPreset (camelcase without italic)

Reply:
fixed in https://github.com/kubernetes/website/pull/14394/commits/bb4445855c8266473904a9566e5791344b91e11d